tools/operator/templates/docker/docker-compose.yaml

```diff
@@ -118,6 +118,7 @@
 
 #@ def orderers(config):
 #@   (ordererUniquerPort, ordererHealthCheckPort) = (30000, 30100)
+#@   env.append({"name": "ORDERER_KAFKA_VERSION", "value": "{}".format(config.kafkaVersion)})
```

Comment:
This is not correct format for docker-compose. Also env is defined in line 125, so this env.append in line 121 will error out.